pkg/kepctl/query_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+
+TO run the test case: go test -v query_test.go create.go  kepctl.go  promote.go query.go
+
+*/
+
+package kepctl
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Tool choice nit: prefer `require` instead of `assert`
Example usage: https://github.com/kubernetes/release/blob/dd16298a1a178a937e02c69a69d6e2c15484fc4e/pkg/anago/anago_test.go#L23

Reply:
Done